nemo-cli/src/cli.rs

```diff
@@ -103,6 +103,12 @@ pub(crate) struct OutputArgs {
     /// does not affect export directives that already specify a compression
     #[arg(short, long = "gzip", default_value = "false")]
     gz: bool,
+    /// Specify which predicates to dump
+    #[arg(long = "dump", value_enum)]
+    pub(crate) dumping: Option<Dumping>,
+    /// File to dump the specified facts
+    #[arg(long = "dump-file", requires = "dumping")]
+    pub(crate) dump_file: Option<PathBuf>,
```

Comment:
I don't think that this is needed as a user could simply redirect stdout to a file.